airbyte-integrations/connectors/source-faker/CHANGELOG.md

```diff
@@ -1,3 +0,0 @@
-| Version | Date       | Pull Request                                             | Subject                     |
-| :------ | :--------- | :------------------------------------------------------- | :-------------------------- |
```

Comment:
Why delete the change log?

Reply:
This was a duplicate - it lives in the connector's doc now